sdcm/nemesis.py

```diff
@@ -1364,6 +1453,16 @@ def disrupt(self):
         self.disrupt_restart_with_resharding()
 
 
+class TopPartitions(Nemesis):
+
+    disruptive = False
+    run_with_gemini = True
+
+    @log_time_elapsed_and_status
+    def disrupt(self):
+        self.disrupt_show_toppartitions()
+
+
```

Comment:
Add to `ChaosMonkey`.
@roydahan which nemesis group we should add this Nemesis?

Reply:
discussed with Roy, please add to ChaosMonkey

Reply:
@bentsi to LimitedChaosMonkey you mean? because in ChaosMonkey it will run any way

Reply:
Added to LimitedChaosMonkey